docs/how-tos/sensor-module.md

```diff
@@ -143,71 +160,28 @@ Run your test script from your terminal and make sure you are able to get readin
-   npm install -g yo
+   viam module generate
    ```
 
-1. Now install the module generator:
-
-   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-   npm install -g generator-viam-module
-   ```
-
-{{% /tablestep %}}
-{{% tablestep %}}
-**2. Run the generator**
-
-Go to the directory where you want to create your module and run the generator:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-yo viam-module
-```
-
-{{% /tablestep %}}
-{{% tablestep %}}
-**3. Name your model**
-
-When prompted for a model triplet, use `<your organization public namespace>:<repo name>:<what you want to call your sensor model>`.
-For example, `jessamy:weather:meteo_PM`.
-
-- You can find your organization namespace by going to your organization settings in the [Viam app](https://app.viam.com).
-- The repo name (also called family name) is generally the name of the GitHub repo where you will put your module code.
-  Name it something related to what your module does.
-- Name your sensor based on what it supports, for example, if it supports a model of ultrasonic sensor called "XYZ Sensor 1234" you could call your model `XYZ_1234` or similar.
-
-For more information, see [Name your new resource model](/how-tos/create-module/#name-your-new-resource-model).
-
-{{% /tablestep %}}
-{{% tablestep %}}
-**4. Specify the API**
-
-For the API triplet, enter `rdk:component:sensor`.
-This means that you are implementing the standard Viam sensor API.
-
-When asked whether this is a Viam SDK built-in API, enter `yes`.
+2. Follow the prompts, naming your module and selecting from the options.
 
-{{% /tablestep %}}
-{{< /table >}}
-<br>
+<!--prettier-ignore-->
+| Prompt | Description |
+| -------| ----------- |
+| Module name | The module name (also called repo name or family name) is generally the name of the GitHub repo where you will put your module code. Name it something related to what your module does. For example, `weather`. |
+| Language | Choose Python to follow this tutorial. |
+| Visibility | Choose `Private` to share only with your organization, or `Public` to share publicly with all organizations. |
+| Namespace/Organization ID | In the [Viam app](https://app.viam.com), navigate to your organization settings through the menu in upper right corner of the page. Find the **Public namespace** and copy that string. In the example snippets below, the namespace is `jessamy`. |
+| Resource to add to the module (API) | Choose `Sensor Component` for this tutorial. |
+| Model name | Name your sensor based on what it supports, for example, if it supports a model of ultrasonic sensor called “XYZ Sensor 1234” you could call your model `XYZ_1234` or similar. |
+| Enable cloud build | You can select `No` for this tutorial because you'll build the module yourself before uploading it. If you select `Yes`, the module will build from your specified GitHub repo using GitHub actions. |
```

Comment:
What specified GitHub repo? If you select yes it doesn't ask you to specify one and you don't need to specify one either. To get this to work you'd need to push it to a GitHub repo and create a release.

Reply:
That is correct. For the purposes of this guide I wanted to keep it simple while eluding to the other option.
```suggestion
| Enable cloud build | You can select `No` for this guide because you'll build the module yourself before uploading it. If you select `Yes`, the module will use GitHub actions to build the module from a GitHub repo you create and specify in the <file>meta.json</file>. |
```

Reply:
Right but you're alluding to it with wrong information that will potentially confuse the reader. The generated .github folder is what will make github automatically use actions once the files are pushed to github and a release is created (which is what I said above). It works like our github actions defined in our .github folder in the docs. I don't think you'd need to specify a GitHub repo in the meta.json at all. The fact that the generator doesn't ask you to and doesn't generate a placeholder for it is an issue because that means our registry won't be able to link to the repo but that is unrelated. It is feedback we should discuss with the CLI team though.

```suggestion
| Enable cloud build | You can select `No` for this tutorial because you'll build the module yourself before uploading it. If you select `Yes` and push the generated files (including the <FILE>.github</FILE> folder) and create a release of the format `vX.X.X`, the module will build and upload to the Viam registry. |
```

Reply:
Ah so it's pushed from the repo, not pulled by the registry. I see. 
(Changing to "guide")
```suggestion
| Enable cloud build | You can select `No` for this guide because you'll build the module yourself before uploading it. If you select `Yes` and push the generated files (including the <file>.github</file> folder) and create a release of the format `vX.X.X`, the module will build and upload to the Viam registry. |
```